### PR TITLE
Week 1 Homework Bilal Goktas

### PR DESCRIPTION
**Pull request by bilalgoktas**

### CODE REVIEW

File: meetupdb.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+connection.query('CREATE DATABASE meetupdb', function (error, results, fields) {
+  if (error) throw error;
+  console.log('"meetupdb" database created.');
+});
+
+connection.query('USE meetupdb', function (error, results, fields) {
+    if (error) throw error;
+  });
+
+connection.query('CREATE TABLE Invitee (invitee_no INT, invitee_name TEXT, invited_by TEXT)', (error, results, fields) => {
```
Comment (acadavid) [inline meetupdb.js:27]: `TEXT` is a type for very large texts. Invitee names are likely to be people names, which would hardly be more than 100 characters. A better type for this is `VARCHAR`. Please update the PRs correcting this type for all tables.

Also I see you added a primary key to some tables. Might as well just add it too all of them! 💪

File: meetupdb.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    if (error) throw error;
+    console.log('"Invitee" table created with "invitee_no" "invitee_name" "invited_by" columns.');
+  });
+
+  connection.query('CREATE TABLE Room (room_no INT, room_name TEXT, floor_number INT, PRIMARY KEY (room_no))', (error, results, fields) => {
```
Comment (acadavid) [inline meetupdb.js:32]: Check comment above about `TEXT` type.

File: meetupdb.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    if (error) throw error;
+    console.log('"Room" table created with "room_no" "room_name" "floor_number" columns.');
+  });
+
+  connection.query('CREATE TABLE Meeting (meeting_no INT, meeting_title TEXT, starting_time DATETIME, ending_time DATETIME, room_no INT, FOREIGN KEY (room_no) REFERENCES Room(room_no))', (error, results, fields) => {
```
Comment (acadavid) [inline meetupdb.js:37]: Nice touch adding  a foreign key

File: worlddb.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    if (error) throw error;
+    console.log("The top 10 most populated cities are as follows relatively: " + results.map(result => result.name))
+  }); 
+
+connection.query("SELECT SUM (population) FROM country", (error, results, fields) => {
+    if (error) throw error;
+    console.log("The total population of the world is: " + results.map((result) => result['SUM (population)']))
+  }); 
+
+connection.end();
```
Comment (acadavid) [inline worlddb.js:71]: All these queries are correct. Well done!

**Review by acadavid — CHANGES_REQUESTED** [review]
@bilalgoktas Good job. Please correct the type of the fields from `TEXT` to `VARCHAR` and I will approve your PR.

**Comment by bilalgoktas** [comment]
> 

@acadavid Thanks a lot for valuable feedback. I have made the necessary changes accordingly.

**Review by acadavid — APPROVED** [review]

**Pull request closed without merging**